guidelines/diagrams/mig-export.txt

```diff
@@ -1,38 +1,66 @@
-# Created at https://sequencediagram.org/
-title MONNAI Informatics Gateway - Export
+#https://sequencediagram.org/
+title Export Results
 
-participant "MIG Export Watcher" as EW
-participant "MONAI Workload Manager\nExport API" as EA
+participantgroup #00CC66 Informatics\nGateway
+participant "SCU Export Watcher" as EW
+participant "SCU Export" as SCU
+participant "FHIR Export" as FHIR
+participant "DICOMweb Export" as DICOMWEB
+
+end 
+
+participantgroup #6699FF Workflow\nManager
+participant "Export API" as EA
+end 
+
+participantgroup #FF9966 Virtual\nStorage
+fontawesome f0a0 "Storage" as DISK
+database "Database" as DB
+end
+
+participantgroup #FFFF99 Pub-Sub\nService
+participant "PubSub" as PS
+end
 
-participant "MIG FHIR Service" as FHIR
-participant "MIG SCU Service" as SCU
-database "MIG\nDatabase" as DB
 fontawesome f0f8 "Hospital\nInformation Systems" as HIS
 
 
+EW->PS: Subscribe to new export task events
+activate EW
+EW<--PS: Response
+
+PS->EW: Notify new export task event
 activate EW
-EW->EA: Query for pending export tasks
+
+EW->EA: Download results

```

Comment:
@mocsharp I was under the impression that the MIG will talk directly to the storage layer to download payloads (having been given a reference in the export event notification) - and not go through the MWM.

Reply:
agree on this point

Reply:
Good catch! Thanks